docs/data/base/getting-started/quickstart/quickstart.md

```diff
@@ -10,24 +10,18 @@ If you're using Next.js 13.4 or later, check out the [Next.js App Router guide](
 
 `@mui/base` is completely standalone – run one of the following commands to add Base UI to your React project:
 
-:::info
-The `next` tag is used to download the latest <b>pre-release</b>, v6 version. Remove it to get the current stable version.
-:::
-
-<!-- #default-branch-switch -->
```

Comment:
@mnajdova Should we keep this for future alpha?

Reply:
We can comment it and keep it